src/languages/jsx.coffee

```diff
@@ -19,6 +19,7 @@ module.exports = {
   ###
   extensions: [
     "jsx",
+    "tsx",
```

Comment:
I wonder if adding to `TypeScript` language would make more sense and have better support via the TypeScript Formatter:  https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/typescript-formatter.coffee#L8